corehq/toggles.py

```diff
@@ -730,11 +730,13 @@ def _ensure_valid_randomness(randomness):
 )
 
 
-ROLE_WEBAPPS_PERMISSIONS = StaticToggle(
-    'role_webapps_permissions',
-    'ICDS: Toggle which webapps to see based on role',
+APP_ACCESS_PERMISSIONS = StaticToggle(
+    'app_access_permissions',
+    'ICDS: Toggle applications that a role would have access to',
     TAG_CUSTOM,
     namespaces=[NAMESPACE_DOMAIN],
+    description=("If this toggle is enabled for a domain then the users will not be allowed to log in if "
+    "they will have no role attached or their role is not configured to access a particular app"),
```

Comment:
This would now say that the user won't be able to access the app on both webapp and phone?

nit: `Toggle applications that a role would have access to`
Not sure if should say "toggle" since that is mainly like a switch on one thing, but this is a collection. May be say "Permit roles to access specific applications only"